Index: components/turbopower_ipro/ipcss.inc
===================================================================
--- components/turbopower_ipro/ipcss.inc	(Revision 44803)
+++ components/turbopower_ipro/ipcss.inc	(Arbeitskopie)
@@ -669,7 +669,7 @@
 function TCSSReader.FindStatement(out AStatement: String): Boolean;
 var
  Buf: char;
- Buf1: array[0..1023] of char;
+ Buf1: string;
  RCount: Integer;
  FStart, FEnd: Integer;
 begin
@@ -687,7 +687,8 @@
     if (Buf = '/') and CheckIsComment then
     begin
       FStream.Position := FStart;
-      RCount := FStream.Read(Buf1[0], FEnd-FStart-1);
+      setlength(Buf1,FEnd-FStart);
+      RCount := FStream.Read(Buf1[1], FEnd-FStart-1);
       AStatement := AStatement + Copy(Buf1,0,RCount);
       FStream.Position := FEnd+1;
       EatComment;
@@ -697,7 +698,8 @@
     begin
       Result := True;
       FStream.Position := FStart;
-      RCount := FStream.Read(Buf1[0], FEnd-FStart);
+      setlength(Buf1,FEnd-FStart);
+      RCount := FStream.Read(Buf1[1], FEnd-FStart);
       AStatement := AStatement + Copy(Buf1,0,RCount);
       break;
     end;
