Index: components/turbopower_ipro/iphtml.pas
===================================================================
--- components/turbopower_ipro/iphtml.pas	(Revision 42366)
+++ components/turbopower_ipro/iphtml.pas	(Arbeitskopie)
@@ -16869,8 +16869,6 @@
 
 procedure TIpHtmlPropA.SetKnownSizeOfSpace(const Size: TSize);
 begin
-  if Size.cx = 0 then
-    raise EIpHtmlException.Create(SHtmlInternal);              {!!.02}
   FKnownSizeOfSpace := Size;
   FSizeOfSpaceKnown := True;
 end;
