Index: src/dbc/ZDbcInterbase6Metadata.pas
===================================================================
--- src/dbc/ZDbcInterbase6Metadata.pas	(Revision 1638)
+++ src/dbc/ZDbcInterbase6Metadata.pas	(Arbeitskopie)
@@ -457,7 +457,8 @@
     'RAW_PARTITIONS,RESERV,RESERVING,RETAIN,RETURNING_VALUES,RETURNS,' +
     'SEGMENT,SHADOW,SHARED,SINGULAR,SNAPSHOT,SORT,STABILITY,STARTS,' +
     'STARTING,STATISTICS,SUB_TYPE,SUSPEND,TRIGGER,VARIABLE,RECORD_VERSION,' +
-    'WAIT,WHILE,WORK';
+    'WAIT,WHILE,WORK,VALUE,POSITION,USER,CURRENCY,OPTION,DATE,START,END,USER,' +
+    'READ,PARENT,TYPE';
 end;
 
 {**
Index: src/plain/ZPlainFirebirdDriver.pas
===================================================================
--- src/plain/ZPlainFirebirdDriver.pas	(Revision 1638)
+++ src/plain/ZPlainFirebirdDriver.pas	(Arbeitskopie)
@@ -89,6 +89,7 @@
 
   WINDOWS20_DLL_LOCATION   = 'fbclient20.dll';
   WINDOWS20_DLL_LOCATION_EMBEDDED = 'fbclientd20.dll';
+  LINUX2_DLL_LOCATION2   = 'libfbclient'+SharedSuffix+'.2';
   LINUX20_DLL_LOCATION   = 'libfbclient'+SharedSuffix+'.20';
   LINUX20_DLL_LOCATION_EMBEDDED = 'libfbembed'+SharedSuffix+'.20';
   LINUX20_IB_CRYPT_LOCATION = 'libcrypt'+SharedSuffix+'.20';
@@ -1268,6 +1269,7 @@
       FLoader.AddLocation(LINUX2_DLL_LOCATION);
     {$ENDIF}
     FLoader.AddLocation(LINUX20_DLL_LOCATION);
+    FLoader.AddLocation(LINUX2_DLL_LOCATION2);
     {$IFDEF ENABLE_INTERBASE_CRYPT}
       {$IFNDEF FIREBIRD_STRICT_DLL_LOADING}
         FPreLoader.AddLocation(LINUX2_IB_CRYPT_LOCATION);
@@ -1386,6 +1388,7 @@
       FLoader.AddLocation(LINUX2_DLL_LOCATION);
     {$ENDIF}
     FLoader.AddLocation(LINUX21_DLL_LOCATION);
+    FLoader.AddLocation(LINUX2_DLL_LOCATION2);
     {$IFDEF ENABLE_INTERBASE_CRYPT}
       {$IFNDEF FIREBIRD_STRICT_DLL_LOADING}
         FPreLoader.AddLocation(LINUX2_IB_CRYPT_LOCATION);
@@ -1515,6 +1518,7 @@
       FLoader.AddLocation(LINUX2_DLL_LOCATION);
     {$ENDIF}
     FLoader.AddLocation(LINUX25_DLL_LOCATION);
+    FLoader.AddLocation(LINUX2_DLL_LOCATION2);
     {$IFDEF ENABLE_INTERBASE_CRYPT}
       {$IFNDEF FIREBIRD_STRICT_DLL_LOADING}
         FPreLoader.AddLocation(LINUX2_IB_CRYPT_LOCATION);
Index: src/plain/ZPlainSqLiteDriver.pas
===================================================================
--- src/plain/ZPlainSqLiteDriver.pas	(Revision 1638)
+++ src/plain/ZPlainSqLiteDriver.pas	(Arbeitskopie)
@@ -566,6 +566,7 @@
     FLoader.AddLocation(WINDOWS_DLL_LOCATION);
   {$ELSE}
     FLoader.AddLocation(LINUX_DLL_LOCATION);
+    FLoader.AddLocation(LINUX_DLL_LOCATION+'.0');
   {$ENDIF}
 end;
 
@@ -1015,6 +1016,7 @@
     FLoader.AddLocation(WINDOWS_DLL3_LOCATION);
   {$ELSE}
     FLoader.AddLocation(LINUX_DLL3_LOCATION);
+    FLoader.AddLocation(LINUX_DLL3_LOCATION+'.0');
   {$ENDIF}
   LoadCodePages;
 end;
